pandas/core/indexes/datetimes.py

```diff
@@ -1937,15 +1937,18 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        DatetimeIndex.tz_localize used for Localize tz-naive DatetimeIndex.
```

Comment:
We should try to avoid to repeat the name of the method in the beginning of this summary line. What do you think of "Localize a tz-naive DatetimeIndex" ? 
At the same time, we could try to make this even clearer. For example use "timezone" instead of "tz".

Reply:
okey sure